include/openthread/srp_server.h

```diff
@@ -365,6 +387,15 @@ void otSrpServerHandleServiceUpdateResult(otInstance *aInstance, otSrpServerServ
  */
 const otSrpServerHost *otSrpServerGetNextHost(otInstance *aInstance, const otSrpServerHost *aHost);
 
+/**
+ * This function returns the response counters of the SRP server.
+ *
+ * @param[in]  aInstance  A pointer to an OpenThread instance.
+ *
+ * @return  A pointer to the response counters of the SRP server.
```

Comment:
```suggestion
 * @return  A pointer to the response counters of the SRP server.
```
Can users get the pointer once and reuse it multiple times? Not sure if we need to document this behavior, or explicitly prohibit.

Reply:
Technically users should be able to reuse it because it's pointing to a member of `Srp::Server`. 

Regarding the comment, I was following the style of `otThreadGetIp6Counters`. 
```
/**
 * Get the IPv6 counters.
 *
 * @param[in]  aInstance  A pointer to an OpenThread instance.
 *
 * @returns A pointer to the IPv6 counters.
 *
 */
const otIpCounters *otThreadGetIp6Counters(otInstance *aInstance);
```

Reply:
SGTM.